web_src/js/features/org-member.js

```diff
@@ -0,0 +1,31 @@
+export function initOrgMemberInvite() {
```

Comment:
It needs a new backend handler (eg: `/org/teams/-/add-member`), the JS code is an over-engineering here.

Reply:
> It needs a new backend handler (eg: `POST /org/members/invite`), the JS code is an over-engineering here.

Let me find some time and try to modify it.